azure-mgmt-web/azure/mgmt/web/models/app_service_plan.py

```diff
@@ -115,11 +116,11 @@ class AppServicePlan(Resource):
         'worker_tier_name': {'key': 'properties.workerTierName', 'type': 'str'},
         'status': {'key': 'properties.status', 'type': 'StatusOptions'},
         'subscription': {'key': 'properties.subscription', 'type': 'str'},
-        'admin_site_name': {'key': 'properties.adminSiteName', 'type': 'str'},
```

Comment:
@Nking92 @panchagnula, assuming this is on purpose, when you update cli, please remove `az appservice plan update --admin_site_name`, tag it with a breaking change, and do call out the reason which users would understand.

Reply:
yes Yugang, will make sure to update this for CLI.